core/src/main/java/org/apache/carbondata/core/cache/dictionary/ReverseDictionaryCache.java

```diff
@@ -43,7 +43,7 @@
    * Attribute for Carbon LOGGER
    */
   private static final LogService LOGGER =
-      LogServiceFactory.getLogService(ForwardDictionaryCache.class.getName());
+      LogServiceFactory.getLogService(ReverseDictionaryCache.class.getName());
```

Comment:
yeah, there is no need to fix this in the current issue. I just found it when I submitted the code...

so, should I rollback this change and start a new issue, OR just add another comment in this issue about this change?